packages/client/e2e/credit.spec.ts

```diff
@@ -1,32 +1,36 @@
-import { fin } from "@openfin/core"
 import { BrowserContext, expect, Page } from "@playwright/test"
 
 import { test } from "./fixtures"
+import {
+  CreditBlotterPageObject,
+  CreditNewRfqPageObject,
+  CreditRfqTilesPageObject,
+  CreditSellSidePageObject,
+} from "./pages"
 import { ExpectTimeout, isOpenFin } from "./utils"
 
-test.describe("Credit", () => {
-  let browserContext: BrowserContext
-  let newRfqPage: Page
-  let rfqsPage: Page
-  let rfqBlotterPage: Page
+let browserContext: BrowserContext
+let mainPage: Page
+let newRfqPage: CreditNewRfqPageObject
+let rfqsPage: CreditRfqTilesPageObject
+let rfqBlotterPage: CreditBlotterPageObject
 
+test.describe("Credit", () => {
   test.beforeAll(async ({ creditPagesRec }, workerInfo) => {
-    newRfqPage = creditPagesRec["credit-new-rfq"]
-    rfqsPage = creditPagesRec["credit-rfqs"]
-    rfqBlotterPage = creditPagesRec["credit-blotter"]
+    mainPage = creditPagesRec.mainPage
+    newRfqPage = creditPagesRec.newRfqPO
+    rfqsPage = creditPagesRec.rfqsPO
+    rfqBlotterPage = creditPagesRec.blotterPO
 
     if (isOpenFin(workerInfo)) {
-      const mainWindow = creditPagesRec["mainWindow"]
-
-      await mainWindow.evaluate(async () => {
-        fin.Window.getCurrentSync().maximize()
+      await mainPage.evaluate(async () => {
+        // eslint-disable-next-line
+        // @ts-ignore
+        window.fin.Window.getCurrentSync().maximize()
       })
-
-      newRfqPage.setViewportSize({ width: 1280, height: 1024 })
-      rfqsPage.setViewportSize({ width: 1280, height: 1024 })
-      rfqBlotterPage.setViewportSize({ width: 1280, height: 1024 })
+      rfqsPage.page.setViewportSize({ width: 1280, height: 1024 })
     } else {
-      await newRfqPage.goto(`${process.env.E2E_RTC_WEB_ROOT_URL}/credit`)
+      await mainPage.goto(`${process.env.E2E_RTC_WEB_ROOT_URL}/credit`)
```

Comment:
is this something we can push back into `fixtures.ts` then we would not need OpenFin conditional in the test ..

Reply:
would need to extract the Sell Side page close below as well ..